css/src/components/gradient-card.scss

```diff
@@ -0,0 +1,34 @@
+$gradient-card-border-radius: $border-radius-lg !default;
+$gradient-card-padding: $layout-2 !default;
+$gradient-card-icon-inline-spacing: 0.375em !default;
+$gradient-card-icon-block-spacing: 0.2em !default;
+
+.gradient-card {
+	border: $border-width-md solid transparent;
+	border-radius: $gradient-card-border-radius;
+	background: linear-gradient(90deg, $gradient-vivid-start, $gradient-vivid-end) border-box;
+	overflow: hidden;
+
+	.gradient-card-content {
+		padding: $gradient-card-padding;
+		background-color: $body-background;
+
+		.gradient-card-title {
```

Comment:
Do we need gradient card title at all here? Shouldn't we be able to do this with atomics and the gradient text atomic?

Reply:
We're already handling icon alignment with atomics on docs-ui (Currently having to use border-color-info and card until this is available, but the deadline to ship dependent changes in docs-ui is EOD today)